jax/_src/interpreters/mlir.py

```diff
@@ -472,7 +490,7 @@ def __init__(
                                         if cached_call_jaxpr_lowerings is None
                                         else cached_call_jaxpr_lowerings)
     self.override_lowering_rules = override_lowering_rules
-    self.shape_poly_state = shape_poly_state or ShapePolyLoweringState(())
+    self.shape_poly_state = shape_poly_state or ShapePolyLoweringState((), ('cpu',))
```

Comment:
I have some questions about why the default value should be always "CPU" but not jax.default_backend()